README.md

```diff
@@ -1 +1,39 @@
 # 2023fa-420-Xterminators
+
+## 📝 Description
+This project is an implementation of the Spelling Bee game in Java 20, created
+as a requirements of the CSCI 420 course. It allows players to play this word
```

Comment:
requirements -> requirement

Reply:
I originally had "created to fulfill the requirements of". I'll fix that.